packages/react-aria-components/src/DropZone.tsx

```diff
@@ -51,9 +51,28 @@ export interface DropZoneProps extends Omit<DropOptions, 'getDropOperationForPoi
 
 export const DropZoneContext = createContext<ContextValue<DropZoneProps, HTMLDivElement>>(null);
 
+const focusableElements = [
+  'input:not([disabled]):not([type=hidden])',
+  'select:not([disabled])',
+  'textarea:not([disabled])',
+  'button:not([disabled])',
+  'a[href]',
+  'area[href]',
+  'summary',
+  'iframe',
+  'object',
+  'embed',
+  'audio[controls]',
+  'video[controls]',
+  '[contenteditable]'
+];
+
+const FOCUSABLE_ELEMENT_SELECTOR = focusableElements.join(':not([hidden]),') + ',[tabindex]:not([disabled]):not([hidden])';
```

Comment:
Grabbed from FocusScope, happy to export it from that package instead but wasn't sure if we want to expose that + the Tabbable selector equivalent

Reply:
maybe not export the list, but we could have export util function like `isFocusable` in that package that just takes a dom element and returns a boolean.

Reply:
sounds good, will add